setup.cfg

```diff
@@ -0,0 +1,9 @@
+[flake8]
+max-line-length = 100
+exclude = doc,
```

Comment:
Coincidentally I just discovered yesterday the flake8 [per-file-ignores](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores) option that let's you exclude just F401 in `__init__.py` files.